docs/release-notes/2.36.0.md

```diff
@@ -0,0 +1,133 @@
+# Percona Monitoring and Management 2.36.0
+
+| **Release date:** | Mar 21, 2023                                                                                 |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+<!---
+!!! caution alert alert-warning "Important/Caution"
+    Crucial points that need emphasis:
+
+    - Important: A significant point that deserves emphasis.
+    - Caution: Used to mean 'Continue with care'.
+ --->
+
+
+## Release Highlights
+
+### pg_stat_monitor 2.0 support in QAN
+
+We are excited to announce PMM 2.36 now supports [pg_stat_monitor](https://docs.percona.com/pg-stat-monitor/index.html) 2.0 (PGSM 2.0) in QAN, a powerful PostgreSQL query performance monitoring tool. By downloading this update, you will have access to the latest improvements and fixes covered by PGSM2, including:
+
+- Improved internal architecture that results in fewer lock acquisitions and increases performance by approximately 20%.
+- Support for PostgreSQL 15 
+- Enhanced consistency with `pg_stat_statements` so that the `pg_stat_monitor` view has identical column names, columns, and data types as `pg_stat_statements` for every major version of PostgreSQL from versions 11 to 15.
+- A bucket status indication (done vs. current) eliminates the need for the tool to evaluate bucket status and facilitates accurate data display.
+- The generation of a unique ID for a query makes it easier to monitor query planning, execution, and performance regardless of version, database, user, or schema.
+
+For more information on the enhancements in PGSM2, refer to the [full list](https://github.com/percona/pg_stat_monitor/blob/main/RELEASE_NOTES.md).
+
+
+
+### Revamped Advisors page
+
+
+We’ve improved the structure of the **Advisors** page to group Advisors by categories and clearly show the Advisors available for your current subscription. 
+
+With this change, we have renamed all the tabs on the page, and added an option to easily integrate with Percona Platform for extra Advisors for free:
+
+  ![!image](../_images/New_Advisors_page.PNG)
+
+
+### PostgreSQL Advisors executed against all DBs
+
+Starting with this release, PostgreSQL Advisors are now executed against all the available databases (DBs), not only the one connected to PMM. This will provide you with insights about all databases in your PostgreSQL server.
```

Comment:
Some PG checks need only default DB to perform needed operation, some need to fetch data from all DBs in the PG instance.
```suggestion
Starting with this release, PostgreSQL Advisors can check all available databases (DBs), not only the one connected to PMM. This will provide you with insights about all databases in your PostgreSQL server.
```